kafka/consumer/multiprocess.py

```diff
@@ -103,7 +107,8 @@ class MultiProcessConsumer(Consumer):
     def __init__(self, client, group, topic, auto_commit=True,
                  auto_commit_every_n=AUTO_COMMIT_MSG_COUNT,
                  auto_commit_every_t=AUTO_COMMIT_INTERVAL,
-                 num_procs=1, partitions_per_proc=0):
+                 num_procs=1, partitions_per_proc=0,
+                 simple_consumer_options={}):
```

Comment:
A mutable dict as a kwarg has import level sideeffects, it'll be one dict shared across all instances.  Please assign this as `None` and then check inside the function.